### DISCUSSION TITLE
Add-PnPSiteScriptPackage for pnp PowerShell - Anyone get this to work?

### DISCUSSION THREAD

**Discussion by wniemiGHUb** [pnp/powershell#2968] [Q&A]
Has anyone been able to get the Add-PnPSiteScriptPackage command to work?  Documentation is very scarce.  I attempted to combine a simple .json site script file with 1 excel file.  I package them together into a package.zip and attempt to call the command using pnpOnline PowerShell:  Add-PnPSiteScriptPackage -Title "MP Projects Package" -Description "Major Projects" -ContentPath "c:\provision\package.zip" and I receive the same error every time - what am I missing here?  thanks,  Wes.

Add-PnPSiteScriptPackage : Unexpected character encountered while parsing value: P. Path '', line 0, position 0.
At line:1 char:1
+ Add-PnPSiteScriptPackage -Title "MP Projects Package" -Description "M ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Add-PnPSiteScriptPackage], PSInvalidOperationException
    + FullyQualifiedErrorId : InvalidOperation,AddSiteScriptPackage

**Comment by michaellumbuku** [comment]
Any resolution to this? Im experiencing the same issue
